### DISCUSSION TITLE
receiver and sender is same

### DISCUSSION THREAD

**Discussion by 308299160** [belaban/JGroups#564] [Q&A] [answered]
in a cluster,
I config bind_port=58002, and config 'port_range'=1.
After the program has been running for a while, 58003 port is running, sender and receiver is same.
![image](https://user-images.githubusercontent.com/12628451/136890956-27408473-125c-4336-80da-12e3f8c4b525.png)
![image](https://user-images.githubusercontent.com/12628451/136891129-a61819b2-4ad5-4f23-9df0-d1e210f10b1c.png)

other servers is normal.
![image](https://user-images.githubusercontent.com/12628451/136891288-f9ed6357-4b04-4dc4-ac9e-cf709346db7d.png)
![image](https://user-images.githubusercontent.com/12628451/136892366-d7f9894c-65e3-44d3-a74e-7b4114efc60d.png)

why dose this happen?

**Comment by belaban** [comment, marked as answer]
This is a known issue and has been fixed. Which version do you use? I'll look into it after my vacation

**Reply by belaban** [reply]
OK, so this issue was fixed in 3.2.9 (commit 881bf4f) on Dec 13 2012. You must be using an older version (almost 10 years old), which I don't recommend!

**Reply by 308299160** [reply]
Thank you for your answer. 
In fact, I use ehcache-jgroupsreplication:1.7  which uses JGroups. I think that ehcache-jgroupsreplication:1.7 maybe didn't update the JGroups version.

**Reply by belaban** [reply]
OK, so perhaps ehcache should upgrade JGroups to a more recent version then...
